### DISCUSSION TITLE
gNB sends E2 SETUP REQUEST with Interface configured as nG instead of F1

### DISCUSSION THREAD

**Discussion by hoangtuansu** [srsran/srsRAN_Project#261] [General Help]
Hi,

I follow the tutorial here

https://docs.srsran.com/projects/project/en/latest/tutorials/source/flexric/source/index.html

and in the message E2SetupRequest, I see that the value of the field e2nodeComponentInterfaceType is _ng_. Is this expected? Shouldn't it be _f1_?

![image](https://github.com/srsran/srsRAN_Project/assets/75421/36632d82-50c4-4561-b8b2-b148a697a490)

Here is the log and the captured pcap files.

[gnb_e2ap_flexric.zip](https://github.com/srsran/srsRAN_Project/files/13055502/gnb_e2ap_flexric.zip)

[gnb_zmq(1).txt](https://github.com/srsran/srsRAN_Project/files/13055494/gnb_zmq.1.txt)

Regards,

**Comment by pgawlowicz** [comment]
Hi, thanks for pointing it out. Indeed, we are not filling those fields correctly. It is on our to-do list. 
BR,
Piotr
